src/sql/SqlService.ts

```diff
@@ -416,4 +417,24 @@ export class SqlServiceImpl implements SqlService {
             return response.rowPage;
         });
     }
+
+    /**
+     * Used for lazy deserialization of row values.
+     * @param data The data to be deserialized.
+     * @param isRaw True if the row is raw, i.e an {@link SqlRowImpl}; false otherwise, i.e a regular json object. Used to log
```

Comment:
The text can be improved a bit by putting true and false in backticks, capitalizing JSON etc.